app/classes/report/mycoportal.rb

```diff
@@ -1,174 +1,348 @@
 # frozen_string_literal: true
 
+require "haversine"
+
+# Report (a TSV spreadsheet) for exporting Observations to MyCoPortal
+# https://mycoportal.org/
+# https://www.mycoportal.org/portal/api/v2/documentation
+
+# MyCoPortal is built on Symbiota
+# https://symbiota.org/
+# https://biokic.github.io/symbiota-docs/
+# https://github.com/Symbiota/Symbiota
 module Report
-  # Report for exporting Observations to MyCoPortal
-  # https://mycoportal.org/
-  # https://www.mycoportal.org/portal/api/v2/documentation
-  #
-  # MyCoPortal is built on Symbiota
-  # https://symbiota.org/
-  # https://biokic.github.io/symbiota-docs/
-  # https://github.com/Symbiota/Symbiota
   class Mycoportal < TSV
-    # These plus Column labels comprising
-    # Symbiota Standard Field names which are useful for our purposes
+    # http_domain for links to Observations and Images
+    HTTP_DOMAIN = "https://mushroomobserver.org"
```

Comment:
Is there a Railsy way of doing this?
(I want a locally-generated report to have links to the website, rather than local links.)

Reply:
What's not "Railsy" about that?

Reply:
I thought there might be a variable I could access instead of hard-coding it.